src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -57,14 +57,25 @@ public enum LanguageVersion
         /// </list> 
         /// </remarks> 
         CSharp6 = 6,
+
         /// <summary>
         /// C# language version 7.
         /// </summary>
         CSharp7 = 7,
+
+        /// <summary>
+        /// The latest version of the language supported.
+        /// </summary>
+        Latest = int.MaxValue,
```

Comment:
If `Latest = 0` then in VB ( with said feature ).
```VB
(Optional langVersion As version = Version.Latest )`
```
will allowed to be written. Which would reduce the "redundant" bolierplate code.
```VB
(Optional langVersion As Version )
```
essential it would `Optional langVersion As Version = default(of Version)`.